bin/license_scout

```diff
@@ -39,3 +39,5 @@ collector.run
 report = collector.issue_report
 
 puts report
+
+exit 2 if !report.empty?
```

Comment:
@danielsdeleo Yeah, although I'm not sure what the best way would be to do that. One thing we could do is merge, then run an ad-hoc build and revert if it causes any trouble; if anyone knows of a way to run a build using this branch, though, that would be better.